istioctl/pkg/util/configdump/cluster.go

```diff
@@ -29,7 +29,7 @@ func (w *Wrapper) GetDynamicClusterDump(stripVersions bool) (*adminapi.ClustersC
 		return nil, err
 	}
 	dac := clusterDump.GetDynamicActiveClusters()
-	slice.Sort(dac, func(i, j int) bool {
+	sort.Slice(dac, func(i, j int) bool {
```

Comment:
Whats the difference between these two?

Reply:
Doing some reading they appear to be the same. `slice.Sort` was added to core Go libraries as `sort.Slice` AFAICT.

Reply:
Yup; so I'm dumping the third-party one in favor of the impl that moved into the standard library.